GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -79,6 +86,19 @@ NS_ASSUME_NONNULL_BEGIN
                                          scopes:(nullable NSArray *)scopes
                                      completion:(nullable GIDSignInCompletion)completion;
 
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
+                       presentingViewController:(nullable UIViewController *)presentingViewController
+                                      loginHint:(nullable NSString *)loginHint
+                                  addScopesFlow:(BOOL)addScopesFlow
+                               verifyCompletion:(nullable GIDVerifyCompletion)completion;
+
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
+                       presentingViewController:(nullable UIViewController *)presentingViewController
+                                      loginHint:(nullable NSString *)loginHint
+                                  addScopesFlow:(BOOL)addScopesFlow
+                         accountDetailsToVerify:(nullable NSArray *)accountDetailsToVerify
```

Comment:
Separately, I don't think that `accountDetailsToVerify` should be `nullable`. The purpose of this method is to provide a mechanism to verify account details. If we don't need to do that, then I don't think we need to call this method.